core/controllers/acl_decorators.py

```diff
@@ -1822,6 +1823,81 @@ def test_can_edit_skill(self, skill_id, **kwargs):
     return test_can_edit_skill
 
 
+def can_edit_skills(handler):
+    """Decorator to check whether the user can edit multiple skills, which can
+    be independent or belong to a topic.
+
+    Args:
+        handler: function. The function to be decorated.
+
+    Returns:
+        function. The newly decorated function that now also checks if
+            the user has permission to edit skills.
+    """
+    def can_user_edit_skill(user, skill_rights):
+        """Checks whether the user can edit the given skill.
+
+        Args:
+            user: UserActionsInfo. Object having user id, role and actions for
+                given user.
+            skill_rights: SkillRights or None. Rights object for the given
```

Comment:
This (the "or None" bit) is an odd construct. Try not to do that; have the caller handle the None case, so that the "skill_rights" passed into here is really a SkillRights object.

Reply:
Done.